### ISSUE TITLE
Does vLLM support the 4-bit quantized version of the Mixtral-8x7B-Instruct-v0.1 model downloaded from Hugging Face

### ISSUE THREAD

**Issue by leockl** [vllm-project/vllm#3128]
Hey guys,

Does vLLM support the 4-bit quantized version of the Mixtral-8x7B-Instruct-v0.1 model downloaded from Hugging Face here https://huggingface.co/mistralai/Mixtral-8x7B-Instruct-v0.1.

According to the Hugging Face link above, we can switch over to the 4-bit version using this line of code:
```
+ model = AutoModelForCausalLM.from_pretrained(model_id, load_in_4bit=True, device_map="auto")
```
This is urgent so I would really appreciate any help on this.

**Comment by simon-mo** [comment]
vLLM support GPTQ and AWQ quantization. You can use https://huggingface.co/TheBloke/Mixtral-8x7B-Instruct-v0.1-GPTQ

**Comment by simon-mo** [comment]
I believe the load_in_4bit uses the wonderful bitsandbytes library [https://huggingface.co/docs/bitsandbytes/main/en/index](https://huggingface.co/docs/bitsandbytes/main/en/index). However, it does not perform calibration and might have worse accuracy on the calibrated GPTQ/AWQ variant.

**Comment by ann-lab52** [comment]
Hi @simon-mo I looked at vllm document and also #392 but didn't find any confirm on **bnb** support. Am I missing anything or it just VLLM doesn't support it yet?

**Comment by simon-mo** [comment]
We do not support bitsandbytes. We are recommending GPT-Q equivalent which have same footprint and higher accuracy.

**Comment by nicklausbrown** [comment]
Hi @simon-mo, I'm trying to load the models in the OpenAI server docker image for both 4 and 8-bit (using the 3.3 release, yay!). I'm running into issues using the --dtype flag which is asking to use float16 and warning about casting from bfloat16. Wouldn't a 4bit or 8bit model be using a different datatype like torch.qint8, sorry if I'm confused and wrong on this. I was trying to load Mixtral at 8-bits onto two 4090's which I believe would work in 8-bit according to this [rough VRAM math](https://huggingface.co/blog/mixtral#inference) but fails for fp16 with OOM even with gpu memory utilization set to 1 and tensor parallel size set to 2. The 4-bit fits however, so perhaps I'm wrong.

**Comment by leockl** [comment]
@simon-mo ok great, many thanks for your help!

**Comment by KeremTurgutlu** [comment]
> We do not support bitsandbytes. We are recommending GPT-Q equivalent which have same footprint and higher accuracy.

Would it be easy to implement bitsandbytes support? There are new techniques which allow quantized-aware training such as QLoRA, which might perform even better than data dependent calibration quantization techniques, at least for fine-tuned models.

**Comment by hmellor** [comment]
`bnb` has been requested in #4033

**Comment by epignatelli** [comment]
> vLLM support GPTQ and AWQ quantization. You can use https://huggingface.co/TheBloke/Mixtral-8x7B-Instruct-v0.1-GPTQ

I can only see AWQ support in the docs?

**Comment by hmellor** [comment]
Yes, GPTQ is supported
